main.go

```diff
@@ -100,7 +100,8 @@ func init() {
 	flag.BoolVar(&config.Heartbleed, "heartbleed", false, "Check if server is vulnerable to Heartbleed (implies --tls)")
 	flag.StringVar(&rootCAFileName, "ca-file", "", "List of trusted root certificate authorities in PEM format")
 	flag.IntVar(&config.GOMAXPROCS, "gomaxprocs", 3, "Set GOMAXPROCS (default 3)")
-	flag.BoolVar(&config.FTP, "ftp", false, "Read FTP banners")
+	flag.BoolVar(&config.FTPBanners, "ftp-banners", false, "Read FTP banners")
+	flag.BoolVar(&config.FTPSBanners, "ftps-banners", false, "Read FTP banners and FTPS certificates")
```

Comment:
I'd rather see --ftp and --ftp-authtls where --ftp means "get banners" and passing `--ftp --ftp-authtls` means get banners and do the auth TLS request, and passing `--ftp-authtls` by itself is an error. This is similar to how the SSH module works.